core/src/main/scala/chisel3/internal/Builder.scala

```diff
@@ -461,6 +461,7 @@ private[chisel3] trait NamedComponent extends HasId {
 
 // Mutable global state for chisel that can appear outside a Builder context
 private[chisel3] class ChiselContext() {
+  val sep: String = "_"
```

Comment:
This could use a slightly more descriptive name, e.g. `modulePrefixSeparator`

Reply:
Done.